src/server/req_rescq.c

```diff
@@ -313,6 +313,8 @@ req_confirmresv(struct batch_request *preq)
 	char		*str_time = NULL;
 	extern char	server_host[];
 	int		is_being_altered = 0;
+	char		*tmp_buf = NULL;
+	int		tmp_buf_size = 0;
```

Comment:
This should probably be size_t instead of int.

Reply:
I believe int will do.

Reply:
It will, I'm just being pedantic. Sizes (as they relate to memory) should be stored in size_t variables.

Reply:
even I like to be :). So, changing it!